.code-samples.meilisearch.yaml

```diff
@@ -0,0 +1,25 @@
+landing_getting_started_1: |-
+  # Create an initializer `config/initializers/meilisearch.rb`
```

Comment:
Should we add comments on these frameworks samples? 
I think they are auto-explanatory for someone who is a ruby/rails developer already, and this could be the same for every other lib.

Reply:
I personally like the precision regarding the path of the file. Is it really obvious to anyone this is `meilisearch.rb` and not `meili.rb` or `meili-search.rb`?

Reply:
As you wish then, if you think `Create an initializer` is the best for the user experience, let's simplify it :)